docs/src/app/page.md

```diff
@@ -23,7 +37,13 @@ LaunchFlow enables you to manage your application's infrastructure, secrets, and
 
 ## Framework Integrations
 
-LaunchFlow is framework agnostic and can run wherever Python runs. It also integrates with popular frameworks, allowing you get started faster. See our framework guides for specific examples of how to integrate LaunchFlow into your existing codebase:
+{% callout type="note" %}
+The integrations below are only available for Python applications.
+
+We will add support for more languages in the future.
+{% /callout %}
+
+LaunchFlow provides integrations with popular Python frameworks to make it easy to connect to cloud resources inside your application. These integrations are optional and can be used to remove most of the boilerplate code needed to connect to AWS / GCP services.
```

Comment:
it might be worth calling out that we can still deploy any docker image here. I worry someone would read this and think oh they don't support react or w/e

Reply:
its mentioned in the section above this